spring-kafka/src/test/java/org/springframework/kafka/retrytopic/AsyncCompletableFutureRetryTopicScenarioTests.java

```diff
@@ -68,6 +69,7 @@
  * @since 3.3.0
  */
 
+@Tag("async-retry-flaky-test")
```

Comment:
I’m sorry, I wasn’t clear: we can add tags eventually if these tests will prove to be sporadically failing.

Reply:
My bad 🙇‍♂️ 
Fixed it!